### DISCUSSION TITLE
Conditional row style with problem?

### DISCUSSION THREAD

**Discussion by russocorp** [KevinVandy/mantine-react-table#59] [General]
Hello.
I'm trying change row color using storybook example but not working.
Using https://www.mantine-react-table.dev/?path=/story/styling-style-table-body-rows--conditionally-style-mantine-table-row to change color of rows, but not working.
Even in the example it's not working apparently.

**Comment by KevinVandy** [comment]
Might just have to add ! important. I'll check it out

**Reply by russocorp** [reply]
Hi Kevin, with this i can change color of background line:

`sx: {
    '&>td': { background: row.getValue('id') = 1 ? selRowBgColor : undefined },
}`
